src/codegen.cpp

```diff
@@ -7159,10 +7159,11 @@ extern "C" void jl_init_codegen(void)
     initializeAnalysis(Registry);
 #endif
 
-    InitializeNativeTarget();
-    InitializeNativeTargetAsmPrinter();
-    InitializeNativeTargetAsmParser();
-    InitializeNativeTargetDisassembler();
+    InitializeAllTargets();
```

Comment:
> LLVM isn't particuclarly known for low memory usage and fast compilation (compare to other JIT, not static compilers).

Thanks for highlighting it !! Didn't know this untill now. I thought it would be convenient to have in case Julia has to target many architectures at once, so that in the future the code wouldn't have to change. Could you point me to some contemporary articles that detail this issue ? Most of those I read are at least 2 years old.